tests/unit/type/zset.tcl

```diff
@@ -431,6 +447,11 @@ start_server {tags {"zset"}} {
         }
 
         test "ZRANK/ZREVRANK basics - $encoding" {
+            if {$::force_resp3} {
+                set nullres {_}
+            } else {
+                set nullres {$-1}
+            }
```

Comment:
let's try to resolve that with the implicit translators mechanism.

Reply:
set nullres <resp2 null>
set nullres_withscores <resp2 other null>
if testing_resp3:
   # rewrite these vars